pkg/storaged/lvol-tabs.jsx

```diff
@@ -537,7 +187,7 @@ export const VDOPoolTab = ({ client, lvol }) => {
         return null;
 
     function grow() {
-        lvol_grow(client, vdo_pool_vol, { });
+        grow_dialog(client, vdo_pool_vol, { });
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19400-20230927-134545-de938f8a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)